code/datums/diseases/_MobProcs.dm

```diff
@@ -17,10 +17,9 @@
 	if(HasDisease(D))
 		return FALSE
 
-	if(!(D.infectable_biotypes & mob_biotypes))
```

Comment:
The real issue is there are still a dozen critters that use mob biotypes lists instead of bitflags.
Removing this will just alter and break some features.